admin/install_server.sh

```diff
@@ -21,6 +21,11 @@ echo "client_encoding = utf8" >> "$PG_CONF"
 
 service postgresql restart
 
+# Less compiler
+curl -sL https://deb.nodesource.com/setup | sudo bash -
```

Comment:
Just a guess, but that sounds like you need the `nodejs-legacy` package, which provides a symlink to `node`.